src/modules/70-pipeline/components/PipelineSteps/Steps/JiraCreate/JiraCreate.tsx

```diff
@@ -95,6 +95,28 @@ export class JiraCreate extends PipelineStep<JiraCreateData> {
       }
     }
 
+    if (template?.spec?.fields && template?.spec?.fields.length > 0) {
+      template?.spec?.fields?.forEach((obj, index) => {
```

Comment:
Inside if block `template.spec.fields` will be truthy
```suggestion
      template.spec.fields.forEach((obj, index) => {
```